docs/JOSS/paper.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'pyhf: pure-Python implementation of HistFactory statistical models'
+tags:
+  - Python
+  - physics
+  - high energy physics
+  - statistical modeling
+  - fitting
+  - auto-differentiation
+authors:
+  - name: Lukas Heinrich
+    orcid: 0000-0002-4048-7584
+    affiliation: 1
+  - name: Matthew Feickert^[Corresponding author.]
+    orcid: 0000-0003-4124-7862
+    affiliation: 2
+  - name: Giordon Stark
+    orcid: 0000-0001-6616-3433
+    affiliation: 3
+  - name: Kyle Cranmer
+    orcid: 0000-0002-5769-7094
+    affiliation: 4
+affiliations:
+ - name: CERN
+   index: 1
+ - name: University of Illinois at Urbana-Champaign
+   index: 2
+ - name: SCIPP, University of California, Santa Cruz
+   index: 3
+ - name: New York University
+   index: 4
+date: 5 October 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+Measurements in High Energy Physics (HEP) rely on determining the compatibility of observed collision events with theoretical predictions.
+The relationship between them is often formalised in a statistical model $f(\mathbf{x}|\mathbf{\phi})$ describing the probability of data $\mathbf{x}$ given model parameters $\mathbf{\phi}$.
+Given observed data, the likelihood $\mathcal{L}(\mathbf{\phi})$ then serves as the basis to test hypotheses on the parameters $\mathbf{\phi}$.
+For measurements based on binned data (histograms), the `HistFactory` family of statistical models [@Cranmer:1456844] has been widely used in both Standard Model measurements [@HIGG-2013-02] as well as searches for new physics [@ATLAS-CONF-2018-041].
+`pyhf` is the first pure-Python implementation of the `HistFactory` model specification and implements a declarative, plain-text format for describing `HistFactory`-based likelihoods that is targeted for reinterpretation and long-term preservation in analysis data repositories such as HEPData [@Maguire:2017ypu].
```

Comment:
could remove "first" here. Not saying that there is something prior, but not sure it adds much.